CONTRIBUTING.md

```diff
@@ -263,6 +263,19 @@ request has been approved and all checks have passed, click the
 button. The resulting commit message will be based on the pull request's title and
 description.
 
+### Reverting a pull request
+
+If a merged pull request needs to be undone, for reasons such as breaking the build, the
+standard process is to revert it through the GitHub interface.
+
+To revert a pull request:
+1.  Navigate to the merged pull request on GitHub.
+2.  Click the **Revert** button. This action automatically creates a new branch and a pull request containing the revert commit.
+3.  Edit the pull request title to comply with the [commit message guidelines](#commit-messages). 
```

Comment:
is there a conventional commit for revert? if so can we say that format here explicitly?